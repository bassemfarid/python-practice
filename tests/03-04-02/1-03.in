100
200

50
25

-5
-15
exit
