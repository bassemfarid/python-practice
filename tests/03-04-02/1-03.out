The sum is: 300
The sum is: 75
The sum is: -20
